### PR TITLE
Update all apprenticeships

### PR DESCRIPTION
**Pull request by MTSarwar**

### CODE REVIEW

File: src/Dfc.ProviderPortal.Apprenticeships/Data/StoredProcedures/UpdateRecordStatuses.js [hunk 1]
```diff
@@ -57,7 +57,8 @@ function updateSproc(UKPRN, currentStatus, statusToBeChangedTo) {
     function tryUpdate(document) {
         // DocumentDB supports optimistic concurrency control via HTTP ETag.
         var requestOptions = { etag: document._etag };
-        document.RecordStatus = statusToBeChangedTo;
```
Comment (gunndabad) [inline src/Dfc.ProviderPortal.Apprenticeships/Data/StoredProcedures/UpdateRecordStatuses.js]: This line needs to stay, need to add:
`document.ApprenticeshipLocations.forEach(l => l.RecordStatus = statusToBeChangedTo);`

File: src/Dfc.ProviderPortal.Apprenticeships/Models/SPResponse.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Dfc.ProviderPortal.Apprenticeships.Models
+{
+    public class SPResponse
```
Comment (gunndabad) [inline src/Dfc.ProviderPortal.Apprenticeships/Models/SPResponse.cs]: Is this class used? If not it can go away

Comment (MTSarwar) [reply src/Dfc.ProviderPortal.Apprenticeships/Models/SPResponse.cs]: Yes it being used to return spresponse
 var response = await client.ExecuteStoredProcedureAsync<SPResponse>(UriFactory.CreateStoredProcedureUri(_settings.DatabaseId, collectionId, "UpdateRecordStatuses"), requestOptions, UKPRN, currentStatus, statusToBeChangedTo);

Comment (gunndabad) [reply src/Dfc.ProviderPortal.Apprenticeships/Models/SPResponse.cs]: Doesn't appear to actually be used there...

File: src/Dfc.ProviderPortal.Apprenticeships/Data/StoredProcedures/UpdateRecordStatuses.js (hunk 1 above)
Comment (MTSarwar) [reply src/Dfc.ProviderPortal.Apprenticeships/Data/StoredProcedures/UpdateRecordStatuses.js]: changed

File: src/Dfc.ProviderPortal.Apprenticeships/Models/SPResponse.cs (hunk 2 above)
Comment (MTSarwar) [reply src/Dfc.ProviderPortal.Apprenticeships/Models/SPResponse.cs]: removed

**Pull request merged**